test/deleteEstudio-test.js

```diff
@@ -0,0 +1,195 @@
+      return this.app.stop();
+    }
+  });
+
+  after(() => {
+    //delete database
+    return database.dropDatabase();
+  });
+  /**
+  * Test Delete estudio button with estudio._id
+  *
+  * Test if the crear-estudio button exists in the
+  * application.
+  */
+  it('should see delete estudio button', async function () {
+    const client = this.app.client;
+    // await sleep(500);
+    return client.setValue('#username',config.username)
+      .setValue('#password', config.password)
+      .click('#submit-login')
+      .then(() => {
+        return client.$('#delete-estudio-' + estudioId);
+      })
+      .then((createBtn) => {
+        assert.isNotNull(createBtn.value);
+      });
+  });
+
+  /**
+  * Test delete estudio
+  *
+  * This test will check if a modal shows before deleting an estudio
+  */
+  it('should see modal before deleting an estudio', async function () {
+    const client = this.app.client;
+    return client.setValue('#username',config.username)
+      .setValue('#password', config.password)
+      .click('#submit-login')
+      .click('#delete-estudio-' + estudioId)
+      .then(async () => {
+        await sleep(1000);
+        return client.getText('#modalContentId');
+      })
+      .then((modalText) => {
+        assert.equal(modalText, 'No podras recuperar el estudio después de esta acción.');
+      });
+  });
+  /**
+  * Test delete estudio
+  *
+  * This test will cancel the modal
+  */
+  it('should cancel delete estudio sweetalert2', async function () {
+    const client = this.app.client;
+    return client.setValue('#username',config.username)
+      .setValue('#password', config.password)
+      .click('#submit-login')
+      .click('#delete-estudio-' + estudioId)
+      .waitForVisible('#modalContentId')
+      .click('.swal2-cancel')
+      .then(async () => {
+        await sleep(1000);
+        return client.$('#delete-estudio-' + estudioId);
+      })
+      .then((estudioButton) => {
+        assert.isNotNull(estudioButton.value);
+      });
+  });
+  /**
+  * Test delete estudio
+  *
+  * This test will delete an estudio
+  */
+  it('should delete an estudio change its status', async function () {
+    const client = this.app.client;
+    return client.setValue('#username',config.username)
+      .setValue('#password', config.password)
+      .click('#submit-login')
+      .click('#delete-estudio-' + estudioId)
+      .waitForVisible('#modalContentId')
+      .click('.swal2-confirm')
+      .then(async () => {
+        await sleep(1000);
+        return client.getText('#modalTitleId');
+      })
+      .then(async (modalValue) => {
+        assert.equal(modalValue, '¡Éxito!');
+        await sleep(1000);
+        return client.click('.swal2-confirm');
+      })
+      .then(async () => {
+        await sleep(1000);
+        return client.click('#show-deleted');
+      })
+      .then(async () => {
+        await sleep(1000);
+        return client.isVisible('#delete-estudio-' + estudioId);
```

Comment:
We do not delete estudiso form the DB, we only change its status to 'Eliminado'. That is why I check into the section of 'Eliminados' at the user dashboard for the estudioId.

Do you think it is still necessary to check that in the DB?
@erikiado